cmd/builder/internal/builder/config.go

```diff
@@ -149,7 +149,7 @@ func (c *Config) SetBackwardsCompatibility() error {
 	c.Distribution.RequireOtelColModule = constraint.Check(otelColVersion)
 
 	// check whether confmap factories are supported
-	constraint, err = version.NewConstraint(">= 0.99.0")
+	constraint, err = version.NewConstraint(">= 0.100.0")
```

Comment:
@codeboten, @evan-bradley not sure if you are aware of this bug, but the release action will mess with the backwards compatible constrains that are added at the latest version.